src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -7948,6 +7980,7 @@ namespace WaterThermalTanks {
         using DataHVACGlobals::TimeStepSys;
         using FluidProperties::GetDensityGlycol;
         using FluidProperties::GetSpecificHeatGlycol;
+        using Psychrometrics::RhoH2O;
```

Comment:
I know `using` statements are littered throughout our code, but the preference moving forward is to be explicit in the line of code -- prefer longer lines of code with Psychrometrics::RhoH2O in them, rather than using using statements at the top of a function.  No big deal for now, just mentioning for the future.

Reply:
Got it, I removed this statement and used a long line in the equation instead.